### ISSUE TITLE
Crash server when i teleport from my space station

### ISSUE THREAD

**Issue by Thysium** [zmaster587/libVulpes#7]
Hello,

I encounter a problem with libvulpes, when i teleport from my space station to other dimensions the server crash. I play for months on this map and i never had this problem until today.

https://pastebin.com/XaJKWpPE

Minecraft: 1.10.2
Libvulpes: v0.1.18
Advanced Rocketry: v1.0.17

**Comment by zmaster587** [comment]
That's an interesting one... you've got some kind of infinite recursion going on, I wonder if somehow there is a tile pointing to itself or pointing back and forth.

**Comment by zmaster587** [comment]
Would it be at all possible to send a copy of the region causing the crash and level.dat?

**Comment by Thysium** [comment]
There are so many region files in the format ' r.x.z.mcr ', how to find the right one ?

**Comment by Thysium** [comment]
Waiting for a reply, i updated the mod storage drawers and the chameleon library, i tested again and i had the same error message but with an additional precision:

> at zmaster587.libVulpes.tile.TilePointer.getFinalPointedTile(TilePointer.java:81) ~[TilePointer.class:unspecified]

**Comment by zmaster587** [comment]
Sorry for the long wait, there's  utility here you can enter the coordinates in and it'll give you the right file names http://dinnerbone.com/minecraft/tools/coordinates/

**Comment by Thysium** [comment]
No problem, i hope to make no mistake, the .mcr file is located at " advRocketry\DIM-2\region " folder ?

https://42.meup.org/g4lq8UnC/r.-4.-3.mca
https://42.meup.org/pjQvYaLg/level.dat

**Comment by zmaster587** [comment]
if it's on a spacestation, yes

**Comment by Thysium** [comment]
Sorry i do not understand your request, by pack you ask if I use a modpack?

**Comment by zmaster587** [comment]
yeah, and if not, can you provide a copy of the mods and configs you have?

**Comment by Thysium** [comment]
I'm sorry for the wait.

https://uploadfiles.io/m4yd2

**Comment by zmaster587** [comment]
I hate to ask for anything else, but can you get me the regions surrounding that one too?  It seems the station is on a region border.

The piece that's here looks really nice though

**Comment by Thysium** [comment]
https://42.meup.org/qxiycD4i.zip

**Comment by zmaster587** [comment]
I apologise for taking so long to get back to you.  Also it seems the link has expired unfortunately.

**Comment by zmaster587** [comment]
In addition, I've added an error output here: http://65.78.115.159:8080/job/LibVulpes/job/libVulpes/job/MC1_10/14/ it should prevent the game from crashing and print a block location the error occured at.
